Helper/Order.php

```diff
@@ -113,6 +113,7 @@ class Order extends AbstractHelper
         'paypal' => 'PayPal',
         'afterpay' => 'Afterpay',
         'affirm' => 'Affirm',
+        'braintree' => 'Braintree'
     ];
```

Comment:
Looks like we need to add apple in this const as well after this PR https://github.com/BoltApp/bolt-magento2/pull/1043
@danac-gs Could you please check?